packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text.cypress.ts

```diff
@@ -0,0 +1,314 @@
+import {
+  generateComponentHTML,
+  TagProps,
+  TestFixture,
+} from '../../../fixtures/test-fixture';
+import {addFacet} from '../../facets/facet/facet-actions';
+import {FacetSelectors} from '../../facets/facet/facet-selectors';
+import {addResultList, buildTemplateWithSections} from '../result-list-actions';
+import {
+  assertShouldRenderValues,
+  assertDisplaysXMoreLabel,
+  assertDoesNotDisplayXMoreLabel,
+} from './result-multi-value-text-assertions';
+import {
+  resultMultiValueTextComponent,
+  ResultMultiValueTextSelectors,
+} from './result-multi-value-text-selectors';
+import * as CommonFacetAssertions from '../../facets/facet-common-assertions';
+
+export interface MultiValueTextProps {
+  field?: string | number;
+  'max-values-to-display'?: number;
+}
+
+const addMultiValueText = (
+  props: MultiValueTextProps = {},
+  slot: HTMLElement[] = []
+) => {
+  const multiValueTextEl = generateComponentHTML(
+    resultMultiValueTextComponent,
+    props as TagProps
+  );
+  slot.forEach((el) => multiValueTextEl.appendChild(el));
+  return addResultList(
+    buildTemplateWithSections({bottomMetadata: multiValueTextEl})
+  );
+};
+
+export interface FacetWithResponseProps {
+  facetId: string;
+  field: string;
+  values: string[];
+  selectedValues: string[];
+}
+
+const addFacetWithResponse =
+  ({facetId, field, values, selectedValues}: FacetWithResponseProps) =>
+  (fixture: TestFixture) => {
+    fixture
+      .with(addFacet({'facet-id': facetId, field}))
+      .withCustomResponse((response) => {
+        response.facets = [
+          {
+            facetId,
+            field,
+            indexScore: 0,
+            moreValuesAvailable: false,
+            values: values.map((value) => ({
+              value,
+              numberOfResults: 1337,
+              state: selectedValues.includes(value) ? 'selected' : 'idle',
+            })),
+          },
+        ];
+      });
+  };
+
+describe('Result MultiValueText Component', () => {
+  describe('when not used inside a result template', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .withElement(generateComponentHTML(resultMultiValueTextComponent))
+        .init();
+    });
+
+    it.skip('should remove the component from the DOM', () => {
```

Comment:
probably I might miss when you mention this, but why we skip these tests? is it because of the test itself, or it's because of issue from atomic?

Reply:
This should already be correct with the @ResultContext decorator. Is not not working correctly ?

Reply:
That's correct, I'm not sure why I skipped it.